lib/recurly.rb

```diff
@@ -15,11 +15,13 @@ module Recurly
   require 'recurly/invoice'
   require 'recurly/js'
   require 'recurly/money'
+  require 'recurly/measured_unit'
```

Comment:
Yes me too, I'd rather do a change like that in another commit or another PR though.